custom_components/frigate/const.py

```diff
@@ -7,17 +7,20 @@
 FRIGATE_RELEASE_TAG_URL = f"{FRIGATE_RELEASES_URL}/tag"
 
 # Icons
-ICON_CAR = "mdi:shield-car"
+ICON_BICYCLE = "mdi:bicycle"
+ICON_CAR = "mdi:car"
 ICON_CAT = "mdi:cat"
 ICON_CONTRAST = "mdi:contrast-circle"
 ICON_CORAL = "mdi:scoreboard-outline"
+ICON_COW = "mdi:cow"
 ICON_DOG = "mdi:dog-side"
 ICON_FILM_MULTIPLE = "mdi:filmstrip-box-multiple"
+ICON_HORSE = "mdi:horse"
 ICON_IMAGE_MULTIPLE = "mdi:image-multiple"
 ICON_MOTION_SENSOR = "hass:motion-sensor"
 ICON_MOTORCYCLE = "mdi:motorbike"
 ICON_OTHER = "mdi:shield-alert"
-ICON_PERSON = "mdi:shield-account"
+ICON_PERSON = "mdi:human"
```

Comment:
From:

![image](https://user-images.githubusercontent.com/29582865/180800655-815862c1-5f45-48bb-a40d-1f4bd8ed5fba.png)


To:

![image](https://user-images.githubusercontent.com/29582865/180800499-ffdf2593-011e-473f-80c0-8d02025f5133.png)

To better represent a human body. WDYT?